indico/modules/users/client/js/ICSCalendarLink.jsx

```diff
@@ -0,0 +1,142 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {Button, Dropdown, Icon, Input, Label, Grid, Popup, Header} from 'semantic-ui-react';
+
+import signURL from 'indico-url:core.sign_url';
```

Comment:
Shouldn't that be at the top of the file?

Reply:
Yes, thanks!